test/transpile.jl

```diff
@@ -803,6 +931,22 @@ end
     end
 end
 
+@testset "cast_to_cx: Readout" begin
```

Comment:
Perhaps, include Toffoli in the test name somewhere

Reply:
fixed